### DISCUSSION TITLE
I want to get not only from key but also from other values

### DISCUSSION THREAD

**Discussion by ghost** [deta/cloud-docs#260] [Help]
Currently in DetaBase, you can get only from the key value, but you need to fetch to get from other values.
This requires a reference to the first item, which is not considered very good.
Can you add or suggest other ways to get from other values?

**Comment by jnsougata** [comment]
You can make queries to base.
Check: https://docs.deta.sh/docs/base/sdk/#fetch

can you be more clear what you are doing and what did you mean by `requires a reference to the first item`?
